zkevm-circuits/src/evm_circuit/execution/begin_tx.rs

```diff
@@ -224,6 +227,17 @@ impl<F: Field> ExecutionGadget<F> for BeginTxGadget<F> {
         let gas_left = tx_gas.expr() - intrinsic_gas_cost.expr();
         let sufficient_gas_left = RangeCheckGadget::construct(cb, gas_left.clone());
 
+        let precompile_warm = array_init::array_init(|_| cb.query_bool());
+        for addr in 1..=PRECOMPILE_COUNT {
+            cb.account_access_list_write(
+                tx_id.expr(),
+                addr.expr(),
+                1.expr(),
+                precompile_warm[addr - 1].expr(),
```

Comment:
why need precompile_warm cells here? I think they must be true?

Reply:
does the prev_value always be false?